### ISSUE TITLE
Linux filesystem extended attributes require a "user." prefix

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2081]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)| @wsanchez reported|
|-|-|
|Trac ID|trac#2081|
|Type|enhancement|
|Created|2006-09-12 20:43:41Z|

Linux filesystem extended attributes require a "user." prefix, at least on ext2.  We have no such prefix, which is resulting a permission denied error from the OS.

<details><summary>Searchable metadata</summary>

```
trac-id__2081 2081
type__enhancement enhancement
reporter__wsanchez wsanchez
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web2_dav web2.dav
keywords__ 
time__1158093821000000 1158093821000000
changetime__1158094275000000 1158094275000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_1"></a>|@wsanchez set status to `assigned`|
|-|-|

This is a mirror of https://trac.macosforge.org/projects/calendarserver/ticket/42, which has a fix submitted by agx@...

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_2"></a>|@wsanchez commented|
|-|-|

Tested on acl branch -8, changeset r18092

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_3"></a>|@wsanchez set status to `closed`|
|-|-|

(In [18102]) Use "user." prefix for extended attributes on Linux.
Submitted by agx@...
Reviewed by wsanchez
Fixes [#2081](https://github.com/twisted/twisted/issues/2081)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
